lib/esp_sdk/end_points/base.rb

```diff
@@ -78,8 +78,20 @@ def submit(url, type, options = {})
         @current_record  = ActiveSupport::HashWithIndifferentAccess.new(JSON.load(response.body))
       end
 
+      # Converts the link header into a hash of links
+      #
+      # "<http://test.host/api/v1/custom_signatures?page=2>; rel=\"last\", <http://test.host/api/v1/custom_signatures?page=2>; rel=\"next\""
+      # => { "last" => "http://test.host/api/v1/custom_signatures?page=2",
+      #      "next" => "http://test.host/api/v1/custom_signatures?page=2" }
       def pagination_links(response)
         @page_links = ActiveSupport::HashWithIndifferentAccess.new(JSON.load(response.headers[:link]))
+      rescue JSON::ParserError
+        @page_links = ActiveSupport::HashWithIndifferentAccess.new.tap do |page_links|
+          response.headers[:link].to_s.split(',').each  do |link|
+            /<(?<link>.*)>; rel="(?<key>\w*)"/ =~ link
+            page_links[key] = link
+          end
+        end
```

Comment:
Nevermore here's the info http://tools.ietf.org/html/rfc5988#page-6 is that what it comes back as?